Src/LiquidProjections.Testing/EnumerableExtensions.cs

```diff
@@ -19,10 +19,7 @@ public static IEnumerable<IList<T>> InBatchesOf<T>(this IEnumerable<T> items, in
                 }
             }
 
-            if (batch.Count != 0)
-            {
-                yield return batch;
-            }
+            yield return batch;

```

Comment:
Because an implementation of an event store must provide at least one empty set of transactions, so that the dispatcher can detect subscribers that are ahead of the event store.